eclair-core/src/test/scala/fr/acinq/eclair/db/AuditDbSpec.scala

```diff
@@ -704,6 +705,184 @@ class AuditDbSpec extends AnyFunSuite {
+              statement.executeUpdate("CREATE INDEX transactions_confirmed_timestamp_idx ON audit.transactions_confirmed(timestamp)")
+
+              setVersion(statement, "audit", 10)
+            }
+
+            using(connection.prepareStatement("INSERT INTO audit.path_finding_metrics VALUES (?, ?, ?, ?, ?, ?, ?, ?)")) { statement =>
+              statement.setLong(1, 214000)
+              statement.setLong(2, 345)
+              statement.setString(3, "FAILURE")
+              statement.setLong(4, 520)
+              statement.setTimestamp(5, TimestampSecond(1651053434L).toSqlTimestamp)
+              statement.setBoolean(6, true)
+              statement.setString(7, "experiment-a")
+              statement.setString(8, "03271338633d2d37b285dae4df40b413d8c6c791fbee7797bc5dc70812196d7d5c")
+              statement.executeUpdate()
+            }
+            using(connection.prepareStatement("INSERT INTO audit.path_finding_metrics VALUES (?, ?, ?, ?, ?, ?, ?, ?)")) { statement =>
+              statement.setLong(1, 35000)
+              statement.setLong(2, 43)
+              statement.setString(3, "SUCCESS")
+              statement.setLong(4, 2043)
+              statement.setTimestamp(5, TimestampSecond(1651054567L).toSqlTimestamp)
+              statement.setBoolean(6, false)
+              statement.setString(7, "experiment-b")
+              statement.setString(8, "030c3f19d742ca294a55c00376b3b355c3c90d61c6b6b39554dbc7ac19b141c14f")
+              statement.executeUpdate()
+            }
+          },
+          dbName = PgAuditDb.DB_NAME,
+          targetVersion = PgAuditDb.CURRENT_VERSION,
+          postCheck = connection => {
+            val migratedDb = dbs.audit
+            using(connection.createStatement()) { statement => assert(getVersion(statement, "audit").contains(PgAuditDb.CURRENT_VERSION)) }
+            using(connection.prepareStatement(s"SELECT amount_msat, status, experiment_name, recipient_node_id FROM audit.path_finding_metrics ORDER BY timestamp")) { statement =>
+              val result = statement.executeQuery()
+              assert(result.next())
+              assert(result.getLong(1) == 214000)
+              assert(result.getString(2) == "FAILURE")
+              assert(result.getString(3) == "experiment-a")
+              assert(result.getString(4) == "03271338633d2d37b285dae4df40b413d8c6c791fbee7797bc5dc70812196d7d5c")
+              assert(result.next())
+              assert(result.getLong(1) == 35000)
+              assert(result.getString(2) == "SUCCESS")
+              assert(result.getString(3) == "experiment-b")
+              assert(result.getString(4) == "030c3f19d742ca294a55c00376b3b355c3c90d61c6b6b39554dbc7ac19b141c14f")
+              assert(!result.next())
+            }
+          }
+        )
+      case _: TestSqliteDatabases => ()
+    }
+  }
+
+  test("migrate sqlite audit database v8 -> current") {
```

Comment:
You can remove that test since we removed the sqlite support.

Reply:
Done